clientconn.go

```diff
@@ -142,6 +154,8 @@ func WithWaitForHandshake() DialOption {
 
 // WithWriteBufferSize lets you set the size of write buffer, this determines how much data can be batched
 // before doing a write on the wire.
+// Zero will disable the write buffer such that each write will be on underlying connection.
+// Note: A Send call may not directly translate to a write.
```

Comment:
Please document the default value here (and also for read).

Reply:
Small note here, in fact gRPC allocates buffer of `WriteBufferSize * 2` size, is it expected and maybe we should update documentation?

Reply:
Done.